### PR TITLE
br: declare phony target

### PR DESCRIPTION
**Pull request by fengou1**
### What problem does this PR solve?
phony target is series of rules/shell commands, it is not real target.

Issue Number: close #xxx 

Problem Summary:
make always show a up to date if any folder and file has the same name with build target.

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:
define phony target
How it Works:
phony target is not real target, but it is series of rules which make to follow.
### Check List 

Tests 

- [x] Unit test
- [x] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
Please add a release note, or a 'None' if it is not needed.
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -238,6 +238,8 @@ bench-daily:
 	cd ./session && \
 	go test -run TestBenchDaily --date `git log -n1 --date=unix --pretty=format:%cd` --commit `git log -n1 --pretty=format:%h` --outfile $(TO)
 
+.PHONY: build_br build_lightning build_lightning-ctl
```
Comment (kennytm) [inline Makefile:241]: use the `.PHONY` on line 17?

**Review by YuJuncen — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -238,6 +238,7 @@ bench-daily:
 	cd ./session && \
 	go test -run TestBenchDaily --date `git log -n1 --date=unix --pretty=format:%cd` --commit `git log -n1 --pretty=format:%h` --outfile $(TO)
 
+
```
Comment (kennytm) [inline Makefile:241]:
```suggestion
```

**Review by kennytm — APPROVED** [review]
rest LGTM

**Review by Leavrth — APPROVED** [review]

**Pull request merged**